app/core/src/main/res/values/arrays_account_settings_values.xml

```diff
@@ -1,6 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
+    <integer-array name="account_colors">
+        <item>0xFF039BE5</item>
+        <item>0xFF388E3C</item>
+        <item>0xFFFB8C00</item>
+        <item>0xFFE53935</item>
+        <item>0xFF8E24AA</item>
+        <item>0xFFD81B60</item>
+        <item>0xFF5E35B1</item>
+        <item>0xFF3949AB</item>
+        <item>0xFF00ACC1</item>
+        <item>0xFF00897B</item>
+        <item>0xFF7CB342</item>
+        <item>0xFFF4511E</item>
```

Comment:
Suggestion:
<img width="200" src="https://user-images.githubusercontent.com/5811634/69355793-b75f9d80-0c82-11ea-8f43-ed3f090eaf43.png" />

Not totally happy with it but I think it looks a bit more "ordered"

```
        <item>0xFFFB8C00</item>
        <item>0xFFF4511E</item>
        <item>0xFFE53935</item>
        <item>0xFFD81B60</item>
        <item>0xFF8E24AA</item>
        <item>0xFF5E35B1</item>
        <item>0xFF3949AB</item>
        <item>0xFF039BE5</item>
        <item>0xFF00ACC1</item>
        <item>0xFF7CB342</item>
        <item>0xFF388E3C</item>
        <item>0xFF00897B</item>
```

Reply:
> Another possibility would be to add some more blue colors and add a fourth line of blue shades.

Sounds good.

Reply:
How about those colors? The last one does not fit 100% but I think it is good to have at least one color that is really neutral.

<img width="250" src="https://user-images.githubusercontent.com/5811634/69373826-f999d680-0ca4-11ea-9eb1-4c7bb7b2938a.png" />



```
        <item>0xFFFFB300</item> <!-- Amber 600 -->
        <item>0xFFFB8C00</item> <!-- Orange 600 -->
        <item>0xFFF4511E</item> <!-- Deep orange 600-->
        <item>0xFFE53935</item> <!-- Red 600 -->

        <item>0xFFC0CA33</item> <!-- Lime 600 -->
        <item>0xFF7CB342</item> <!-- Light green 600 -->
        <item>0xFF388E3C</item> <!-- Green 700 -->
        <item>0xFF00897B</item> <!-- Teal 600 -->

        <item>0xFF00ACC1</item> <!-- Cyan 600 -->
        <item>0xFF039BE5</item> <!-- Light blue 600 -->
        <item>0xFF1976D2</item> <!-- Blue 700 -->
        <item>0xFF3949AB</item> <!-- Indigo 600 -->

        <item>0xFFE91E63</item> <!-- Pink 500 -->
        <item>0xFF8E24AA</item> <!-- Purple 600-->
        <item>0xFF5E35B1</item> <!-- Deep purple 600-->
        <item>0xFF455A64</item> <!-- Blue gray 700 -->
```